src/test/java/ch/liquidmind/inflection/compiler/InflectionCompilerTest.java

```diff
@@ -0,0 +1,55 @@
+package ch.liquidmind.inflection.compiler;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import ch.liquidmind.inflection.compiler.CompilationJob.CompilationMode;
+import ch.liquidmind.inflection.compiler.util.CompilerTestUtility;
+
+public class InflectionCompilerTest {
+
+	@Test
+	public void testCompile_ValidFile_SuccessfulCompilation() throws Exception {
+		CompilationJob job = CompilerTestUtility.createCompilationJob(CompilationMode.BOOTSTRAP, 
+				this.getClass().getResource("InflectionCompilerTest_ValidFile.inflect"));
```

Comment:
I would introduce a new utility method "getClassLoader" that returns the class loader to use for testing. I would also recommend using Thread.getContextClassLoader().getResource() instead of Class.getResource; the rationale is that the class loader of the test class might not necessarily be the class loader through which the resource can be located. If we start running the compiler and proxy generator through custom class loaders, this may be relevant.